HISTORY.rst

```diff
@@ -2,6 +2,40 @@
 History
 =======
 
+3.26.0 (2021-01-08)
+-------------------
+
+Critical Changes:
+
+- CumulusCI can now resolve dependencies using second-generation packages (2GPs) for upstream projects. When a 2GP build runs on a release branch (starting with ``prefix/NNN``, where ``prefix`` is the feature branch prefix and ``NNN`` is an integer), CumulusCI will look for a matching release branch in each upstream dependency and use a 2GP package build on that release branch if present, falling back to a 1GP beta release if not present.
```

Comment:
This is just a change, not a critical change. Nothing is different unless you turn on the new option.

Reply:
As implemented it is on by default, which would make it a critical change to me. 

Should we quick-change that to be off by default? That's probably a lot safer. @davisagli @Br4nd0R

Reply:
@davidmreed It appears to already be off by default: https://github.com/SFDO-Tooling/CumulusCI/blob/main/cumulusci/tasks/salesforce/update_dependencies.py#L84

Reply:
It's turned on in `cumulusci.yml` for 2GP builds: https://github.com/SFDO-Tooling/CumulusCI/blob/main/cumulusci/cumulusci.yml#L933

Reply:
Ah ok, that's the disconnect. Can we be more specific and name specific flows rather than vaguely mentioning "2gp builds" in this note?